docs/guides/consul-federation.md

```diff
@@ -10,20 +10,26 @@ description: |-
 Once you have a HCP Consul cluster, you can create a new Consul cluster to federate with the existing one.
 
 ```terraform
-resource "hcp_hvn" "example" {
-  hvn_id         = var.hvn_id
+resource "hcp_hvn" "primary" {
```

Comment:
I'm thinking we could expand this guide. So this example could have the heading `Federation with Auto HVN Peering`, and show example HCL using auto peering. And then we can later add another section with a manual HVN peering example.